src/coreclr/jit/gentree.cpp

```diff
@@ -24073,6 +24073,51 @@ void GenTreeHWIntrinsic::SetHWIntrinsicId(NamedIntrinsic intrinsicId)
            (op1->GetAuxiliaryType() == op2->GetAuxiliaryType()) && (op1->GetOtherReg() == op2->GetOtherReg()) &&
            OperandsAreEqual(op1, op2);
 }
+
+genTreeOps GenTreeHWIntrinsic::HWOperGet()
```

Comment:
@SkiFoD can you please add a standard header to this function? e.g.:
```
//-----------------------------------------------------------
// HWOperGet: %what it does%
//
// Return Value:
//     %what it returns%
//
```

Reply:
@SkiFoD this is a kindly reminder to address this comment and resolve conflict.

> @SkiFoD can you please add a standard header to this function? e.g.:
> 
> ```
> //-----------------------------------------------------------
> // HWOperGet: %what it does%
> //
> // Return Value:
> //     %what it returns%
> //
> ```